doc/classes/SubViewport.xml

```diff
@@ -10,7 +10,7 @@
 	</methods>
 	<members>
 		<member name="render_target_clear_mode" type="int" setter="set_clear_mode" getter="get_clear_mode" enum="SubViewport.ClearMode" default="0">
-			The clear mode when the sub-viewport is used as a render target.
+			The clear mode when the sub-viewport is used as a render target. Note this property is intended for 2D usage.
```

Comment:
```suggestion
			The clear mode when viewport used as a render target.
			[b]Note:[/b] this property is intended for 2D usage.
```

Reply:
I've noticed that in your comment here you're also suggesting to change the first sentence, i.e. replace "the sub-viewport" with "viewport" and remove "is". 
Just wanted to double check if you're sure about the changes or if they were perhaps unintentional :sweat_smile:

Reply:
It was unintentional, yes. I'm fixing it